playbooks/files/create_cts_tracker.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import openstack
+import sys
+
+conn = openstack.connect()
+
+obs_bucket = sys.argv[1]
+
+attrs = {
+    "bucket_name": obs_bucket,
+    "file_prefix_name": "file-apimon-",
+    "lts": {
+        "is_lts_enabled": True
+    },
+    "status": "enabled",
+    "tracker_name": "system",
+    "detail": ""
+}
+
+tracker = conn.cts.create_tracker(**attrs)
```

Comment:
May fail if tracker "system" still exists, for example since the `delete_cts_tracker.py` failed. I sggest to add at least a simple exception handling.

Reply:
I have added some additional error handling. See my latest commit afc4b6efce5360c0164cdfa4744be3b8fb92c37e